### PR TITLE
Improve CLEAR_MODES docs

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
As discussed with @SukantPal , I have to clarify this once more.

Most people that write filters dont know about clamping and other tricks that are needed in case we use pow2 temporary renderTextures. Even our `BlurFilter` is affected, @GoodBoyDigital wrote this version for v5 :)

For them, we distinguish two clearing modes - one `CLEAR` that clears everything, and second `BLIT` that just overwrites new content over texture, and leaves right and bottom zone as it was.

### CODE REVIEW

**Comment by SukantPal** [comment]
@ivanpopelyshev BLIT mode doesn't not clear only the filtering area on the texture right now. If that was the intended purpose, we need to fix it.

**Comment by bigtimebuddy** [comment]
I find this doc change not particularly useful or legible. This need some editing for clarity.

**Review by SukantPal — APPROVED** [review]
Hey @bigtimebuddy, I had a chat with @ivanpopelyshev and we cleared up (get it?) what `CLEAR_MODES.BLIT` should be doing. It basically skips clearing the filter texture if the filter has `state.blend = false` - as that will overwrite pixels anyway. Setting `forceClear` on the `FilterSystem` will make this mode clear regardless of the blend mode. That flag may be used in the future to force clearing on devices where that operation makes things faster.

**Review by SukantPal — APPROVED** [review]
Hey @bigtimebuddy, I had a chat with @ivanpopelyshev and we cleared up (get it?) what `CLEAR_MODES.BLIT` should be doing. It basically skips clearing the filter texture if the filter uses `BLEND_MODES.NONE` - as that will overwrite pixels anyway. Setting `forceClear` on the `FilterSystem` will make this mode clear regardless of the blend mode. That flag may be used in the future to force clearing on devices where that operation makes things faster.

File: packages/core/src/filters/FilterSystem.ts
Comment (GoodBoyDigital) [inline packages/core/src/filters/FilterSystem.ts:346]: heya! would be great to comment this line, as its hard to tell whats going on 🥇

Comment (bigtimebuddy) [reply packages/core/src/filters/FilterSystem.ts:346]: This line:
```js
const autoClear = (stateSystem.stateId & 1) || this.forceClear;
```

**Review by GoodBoyDigital — APPROVED** [review]
lgtm 👍

File: packages/core/src/filters/FilterSystem.ts
Comment (SukantPal) [reply packages/core/src/filters/FilterSystem.ts:346]: I'll add a comment when the workday is over (be patient, don't merge this until I do pls even tho it has 2 approvals)

Comment (bigtimebuddy) [reply packages/core/src/filters/FilterSystem.ts:346]: Roger that!

**Review by bigtimebuddy — APPROVED** [review]

**Comment by bigtimebuddy** [comment]
Thank you @SukantPal and @ivanpopelyshev for making this better.

**Pull request merged**
